### PR TITLE
Match all FX render functions

### PR DESCRIPTION
**Pull request by Brotenko**
Warnings can't be resolved right now due to missing insight on the inner workings of effects, especially RenderTasks and fx_NUM_appendGfx functions.

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 112 functions (+0.24%) on `us`.

⚠️ This PR introduces 112 warnings:
See log for details.

File: src/effects/effect_60.c [hunk 1]
```diff
@@ -7,7 +12,26 @@ void fx_60_init(void) {
 
 INCLUDE_ASM(s32, "effects/effect_60", fx_60_update);
 
-INCLUDE_ASM(s32, "effects/effect_60", fx_60_render);
+void fx_60_appendGfx(EffectInstance* effect);
+void fx_60_render(EffectInstance* effect) {
+    Effect60* effect60 = effect->data;
+    RenderTask renderTask;
+    RenderTask* renderTaskTemp = &renderTask;
```
Comment (ethteck) [inline src/effects/effect_60.c:19]: etc

File: src/effects/effect_66.c [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #include "common.h"
+#include "effects.h"
```
Comment (ethteck) [inline src/effects/effect_66.c:2]: effects_internal.h

File: src/effects/effect_94.c [hunk 3]
```diff
@@ -11,7 +17,25 @@ f32 func_E00BC1D8(f32 arg0) {
...
+void fx_94_appendGfx(EffectInstance* effect);
+void fx_94_render(EffectInstance *effect) {
+    Effect94* effect94 = effect->data;
+    RenderTask renderTask;
+    RenderTask* retTask;
+    RenderTask* renderTaskPointer = &renderTask;
+
+    renderTask.appendGfx = fx_94_appendGfx;
+    renderTask.appendGfxArg = effect;
+    if (gGameStatusPtr->isBattle == 1) {
```
Comment (ethteck) [inline src/effects/effect_94.c:29]: == TRUE

File: src/world/area_mac/mac_04/843030.c [hunk 4]
```diff
@@ -80,7 +80,17 @@ INCLUDE_ASM(s32, "world/area_mac/mac_04/843030", func_80241A64_844634);
 
 INCLUDE_ASM(s32, "world/area_mac/mac_04/843030", func_80241B90_844760);
 
-INCLUDE_ASM(s32, "world/area_mac/mac_04/843030", func_80241BB8_844788);
+ApiStatus N(SetNpcShadowScale)(Evt* script, s32 isInitialCall) {
+    s32 npcID;
+    f32 newShadowScale;
+
+    Bytecode* args = script->ptrReadPos;
```
Comment (ethteck) [inline src/world/area_mac/mac_04/843030.c:87]: args at the top of the func, and then npcID on the same line it's declared, followed by newShadowScale

**Review by ethteck — CHANGES_REQUESTED** [review]
Nice work on these! Mostly just naming moving stuff

**Review by ethteck — APPROVED** [review]

File: src/effects/effect_117.c [hunk 5]
```diff
@@ -7,7 +19,39 @@ void fx_117_init(void) {
 
 INCLUDE_ASM(s32, "effects/effect_117", fx_117_update);
 
-INCLUDE_ASM(s32, "effects/effect_117", fx_117_render);
+void fx_117_appendGfx(EffectInstance* effect);
```
Comment (nanaian) [inline src/effects/effect_117.c:22]: Move this declaration to the top of the file pls

**Review by nanaian — CHANGES_REQUESTED** [review]
Damn thats a lot of functions, gg

**Pull request merged**